sbi/inference/snl/snl.py

```diff
@@ -53,6 +54,10 @@ def __init__(
         perform inference on the posterior p(theta | x0).
         :param neural_likelihood: Conditional density estimator q(x | theta) in the form of an
         nets.Module. Must have 'log_prob' and 'sample' methods.
+        simulation_batch_size (int): how many parameter sets the simulator takes and converts to data x at
+                the same time. If simulation_batch_size==-1, we simulate ALL parameter sets at the same time.
+                If simulation_batch_size==1, the simulator has to process data of shape (num_dim).
```

Comment:
I think it would be best if both if data in the shape `(1, num_dim)` was passed to the simulation if `simulation_batch_size==1`